src/compiler/parse/state/tag.ts

```diff
@@ -9,6 +9,9 @@ import read_script from '../read/script';
 import read_style from '../read/style';
 import { closing_tag_omitted, decode_character_references } from '../utils/html';
 
+// characters equivalent to using \s in regex
+const whitespace_characters = ['\r', '\n', '\t', '\f', '\v', ' ', '\u00a0', '\u1680', '\u2000-', '\u200a', '\u2028', '\u2029', '\u202f', '\u205f', '\u3000', '\ufeff'];
```

Comment:
Probably a mistake.

```suggestion
const whitespace_characters = ['\r', '\n', '\t', '\f', '\v', ' ', '\u00a0', '\u1680', '\u2000', '\u2001', '\u2002', '\u2003', '\u2004', '\u2005', '\u2006', '\u2007', '\u2008', '\u2009', '\u200a', '\u2028', '\u2029', '\u202f', '\u205f', '\u3000', '\ufeff'];
```

Reply:
I got this list of characters from regex101:

<img width="416" alt="Screenshot 2022-07-10 at 09 56 03" src="https://user-images.githubusercontent.com/48158184/178136305-9a97c7b7-8470-43fd-9738-da64e5171b77.png">

I was also unsure about the "-" being correct, so if it is a mistake I'll remove it!

Reply:
You can see the "-" on the end at the \u2000 part, which is why i included it

Reply:
You can learn the meaning of hyphens.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Character_Classes

Reply:
Oh, i did not see all the extra characters you added. Thank you for clearing this up!